### ISSUE TITLE
[Feature] Connect to running browser missing

### ISSUE THREAD

**Issue by fehmer** [microsoft/playwright-java#179]
`BrowserType.connect` is missing from the implementation.

The api defines https://github.com/microsoft/playwright/blob/master/docs/api.md#browsertypeconnectparams but seems to be missing from the java implementation.

Example code (node.js)
```javascript
const pw = require('playwright');

(async () => {

    const browser = await pw.chromium.connect({
        wsEndpoint: 'ws://localhost:3000/playwright',
    });

    const context = await browser.newContext();
    const page = await context.newPage();
    await page.goto('http://whatsmyuseragent.org/');
    await page.screenshot({ path: `example-connect.png` });
    await browser.close();

})();
```

**Comment by mxschmitt** [comment]
Hi, thats currently an upstream limitation and blocked by it: https://github.com/microsoft/playwright/issues/4687

**Comment by ghoshasish99** [comment]
Hi @mxschmitt  , any estimated timeline for this issue?

**Comment by yury-s** [comment]
Connected has been added back for now and is available in v1.9.0-alpha-0.
